src/skmultiflow/meta/learn_nse.py

```diff
@@ -265,11 +270,33 @@ def __vote_proba(self, X, t, classes):
                     h = self.ensemble[i]
                     y_predicts = h.predict_proba(X[m].reshape(1, -1))
                     y_predicts /= np.linalg.norm(y_predicts, ord=1, axis=1, keepdims=True)
-                    votes += self.ensemble_weights[i] * y_predicts
+                    try:
+                        votes += self.ensemble_weights[i] * y_predicts
+                    except ValueError:
+                        # sklearn learner
+                        if hasattr(h, 'classes_'):
+                            obs_classes = h.classes_
+                        else:  # skmultiflow learner
+                            obs_classes = h.classes
+                        votes += self.ensemble_weights[i] * \
+                            self._fill_missing_probs(
+                                y_predicts, obs_classes, self.classes
+                            )
 
             res.append(votes.reshape(len(classes)))
         return np.array(res)
 
+    def _fill_missing_probs(self, probs, obs_classes, all_classes):
+        proba_ordered = np.zeros(
+            (probs.shape[0], all_classes.size), dtype=np.float
+        )
+        sorter = np.argsort(all_classes)
```

Comment:
What about renaming `sorter` as `sorterd_classes` for clarity

Reply:
Totally agreed! I renamed the variable in commit [3ec9def](https://github.com/scikit-multiflow/scikit-multiflow/pull/223/commits/3ec9defdc5593530fffc588be8054e6c091805a0). Thanks for the suggestion @jacobmontiel.